include/ifc/abstract-sgraph.hxx

```diff
@@ -27,6 +27,16 @@
 #include "ifc/operators.hxx"
 #include "ifc/pp-forms.hxx"
 
+#ifdef _MSC_VER
+#define IFC_STRUCT_PACK(...) __pragma(pack(push, 4)) __VA_ARGS__ __pragma(pack(pop))
+#define IFC_STRUCT_PACK_VERIFY(type, size) static_assert(sizeof(type) == (size))
```

Comment:
Let's do without the macro `IFC_STRUCT_PACK_VERIFY`.  Just writing out the `static_assert` would be just fine.

Reply:
Yes, I left it there from a previous attempt.

Reply:
BTW, the previous `#pragma pack` syntax worked with GCC+Clang, right?

Reply:
Oops, it appears they do: https://godbolt.org/z/znjWb5z9M

I was too eager about the warning pragma being unrecognized.

Reply:
No worries.  Let's keep the original form, without the macro.